db/generateData.js

```diff
@@ -25,7 +25,7 @@ const generatePartySize = () => Math.floor(Math.random() * 19 + 1);
 let primaryKey = 1;
 const generateReservations = (restaurantID) => {
   const data = [];
-  for (let day = 0; day < 10; day += 1) {
+  for (let day = 0; day < 1; day += 1) {
```

Comment:
What's the reason for this change?

Reply:
Thanks for catching this, I reduced the size of my data for testing purposes but forgot to revert this!

This value should be 10.